commands/stonks.js

```diff
@@ -7,20 +7,27 @@ api_key.apiKey = config.finnhub_api_key;
 const finnhubClient = new finnhub.DefaultApi();
 
 exports.run = async (client, message, args, level) => {
-  console.log(args);
-  const company = args[0];
-  console.log(company);
-
-  if(company) {
-    finnhubClient.quote("VEEV", (error, data, response) => {
+  if(args[0]) {
+    const company = args[0].toUpperCase();
+    finnhubClient.quote(company, (error, data, response) => {
+      console.log(company);
       console.log(data);
-      message.channel.send(`\`\`\`\n
+      if(Object.keys(data).length === 0) {
+        message.reply("Invalid ticker ["+company+"]");
+      } else {
+        message.channel.send(`\`\`\`\n
+Company: ${company}
+
 Today's Open: ${data.o}
 Today's High: ${data.h}
 Today's Low: ${data.l}
 Current Price: ${data.c}
 Yesterday's Close: ${data.pc}\`\`\``);
+      }
     });
+      
+  } else {
+    message.reply(".stonks [ticker]")
```

Comment:
The `.` is configurable right?

Reply:
True... I wont address this one tho. Too much work.

Reply:
Damn it. You're fired!